en/flight_controller/pixracer.md

```diff
@@ -167,6 +167,18 @@ For information about wiring and using this port see:
 - [PX4 System Console](http://dev.px4.io/master/en/debug/system_console.html#pixhawk_debug_port) (Note, the FMU console maps to UART7).
 
 
+## Serial Port Mapping
+
+UART | Device | Port
+--- | --- | ---
+UART1 | /dev/ttyS0 | WiFi
```

Comment:
@pkocmoud UART 1 formerly said "IO debug (except v4, there ttyS0 is the wifi)". Is this change correct for Pixracer/FMUv4?

See https://github.com/PX4/px4_user_guide/pull/672/files#r387455255

Reply:
You might want to specify ESP8266 WiFi (The ports is a header that is pinned for that device) but otherwise this looks good to me!

Reply:
Thanks!